types/react-bootstrap/lib/Modal.d.ts

```diff
@@ -28,7 +28,8 @@ declare namespace Modal {
         enforceFocus?: boolean;
         keyboard?: boolean;
         onBackdropClick?: (node: HTMLElement) => any;
-        onEscapeKeyUp?: (node: HTMLElement) => any;
+        onEscapeKeyDown?: (node: HTMLElement) => any;
+        onEscapeKeyUp?: (node: HTMLElement) => any; // deprecated, use onEscapeKeyDown
```

Comment:
Can you use a jsdoc comment here:
```
/**
 * @deprecated since {whenever}, use onEscapeKeyDown instead
 **/
onEscapeKeyUp?: (node: HTMLElement) => any;
```
This will give better hints in supported IDEs.

Reply:
sure thing